### PR TITLE
[170] Documentation - Intro and Installation

### PR DESCRIPTION
**Pull request by Aman1905**
Here is the Issue link - https://github.com/StackGuardian/tirith/issues/170 

Added the 3 methods of Installation of Tirith
1. Quick Installation.
2. Developer Installation.
3. Manual Installation. 

- made the grammatical correction as well

### CODE REVIEW

**Review by AkashS0510 — APPROVED** [review]

File: documentation/docs/tirith-installation/developer-mode-installation.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    },{
+      "@type": "ListItem",
+      "position": 2,
+      "name": "Support",
+      "item": "https://www.lambdatest.com/support/docs/"
+    },{
+      "@type": "ListItem",
+      "position": 3,
+      "name": "Installation",
+      "item": "https://www.lambdatest.com/support/docs/developer-mode-installation/"
```
Comment (refeed) [inline documentation/docs/tirith-installation/developer-mode-installation.md:32]: I think we can remove this, also the same for the other md files

Comment (Aman1905) [reply documentation/docs/tirith-installation/developer-mode-installation.md:32]: @refeed i can remove this as of now but later would have to add these again once the deployment will be done

Comment (refeed) [reply documentation/docs/tirith-installation/developer-mode-installation.md:32]: alright, then let's keep it

**Review by refeed — APPROVED** [review]

**Comment by refeed** [comment]
Thanks @Aman1905 !

**Pull request merged**